ci/scripts/r_docker_configure.sh

```diff
@@ -39,6 +39,16 @@ if [ "$RHUB_PLATFORM" = "linux-x86_64-fedora-clang" ]; then
   rm -rf $(${R_BIN} RHOME)/etc/Makeconf.bak
 fi
 
+# Special hacking to try to reproduce quirks on centos using non-default build
+# tooling.
+if [[ "$R_TAG" == *centos* && "$DEVTOOLSET_VERSION" > 0 ]]; then
+  if [ "`which dnf`" ]; then
+    dnf install -y "devtoolset-$DEVTOOLSET_VERSION"
+  else [ "`which yum`" ]; then
```

Comment:
If we have a devtoolset specified, we want it to install it one way or another

```suggestion
  else
```